website_mass_mailing_name/static/src/js/website_mass_mailing_name.js

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+odoo.define("website_mass_mailing_name.subscribe", function (require) {
+    "use strict";
+    require("mass_mailing.website_integration");
+    var animation = require("website.content.snippets.animation");
+
+    animation.registry.subscribe.include({
+        start: function(editable_mode) {
+            this.$email = this.$target.find(".js_subscribe_email");
```

Comment:
Can use this.$() instead of this.$target.find()...

Reply:
Not against but see your review is LGTM but still asking for changes, would be better move forward instead spend more time waiting for Travis?

Reply:
Yeah, without these changes all works fine... it's just a small suggestion that can be completely ignored ;)
** Same for the other comment.